source/features/github-actions-indicators.tsx

```diff
@@ -66,24 +102,28 @@ const getWorkflowsDetails = cache.function('workflows', async (): Promise<Record
 });
 
 async function addIndicators(workflowListItem: HTMLAnchorElement): Promise<void> {
-	// Memoized above
-	const workflows = await getWorkflowsDetails();
-	if (!workflows) {
-		return; // Impossibru, for types only
-	}
-
 	if (select.exists('.octicon-stop', workflowListItem)) {
```

Comment:
It turns out GitHub already had this indicator before, we were event skipping items that had it: https://docs.github.com/en/enterprise-server@3.4/actions/managing-workflow-runs/disabling-and-enabling-a-workflow

![actions-find-disabled-workflow](https://user-images.githubusercontent.com/1402241/234182928-969be1f6-0c3f-4fb4-8223-aac09a4eee5c.jpeg)

For this reason I replaced the Alert icon with a Stop icon

```suggestion
	// There might be a disabled indicator already
	if (select.exists('.octicon-stop', workflowListItem)) {
```